embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/session/orchestrator/V2PayloadStore.kt

```diff
@@ -41,6 +41,17 @@ internal class V2PayloadStore(
         intakeService.take(envelope, createMetadata(type, payloadType = payloadType))
     }
 
+    override fun cacheEmptyCrashEnvelope(envelope: Envelope<LogPayload>) {
+        intakeService.take(
+            intake = envelope,
+            metadata = createMetadata(
+                type = SupportedEnvelopeType.CRASH,
+                complete = false,
```

Comment:
Previously we didn't have a concept of incomplete logs. I haven't spotted anything in the changeset that would concern me but it'd be worth reviewing how log persistence works at a high-level to see if this change breaks any assumptions.

Reply:
Correct. In this patch I made sure things are cleaned up at resurrection time, and there shouldn't be anything that would generically use that payload.

There's coupling in the intake service that I'm not thrilled about with respect to cleaning up the "last" cached object, but it's also the best place since it can confirm when it's actually done. Maybe a call back mechanism should be added to facilitate this, but I think I want to wait until this `PayloadStore` concept is removed before doing that.